.github/workflows/ci-integration.yaml

```diff
@@ -120,7 +129,7 @@ jobs:
           sh -c 'printf "%s\n" "$GOOGLE_APPLICATION_CREDENTIALS_CONTENTS" > "$GCE_CREDENTIALS_PATH"'
 
           # SSH keys (should match what specified in pycloudlib.toml)
-          mkdir ~/.ssh
+          mkdir ~/.ssh || true
```

Comment:
ah good point, I'll change that - this was only useful when using https://github.com/marketplace/actions/debugging-with-ssh to debug because that action had already created the directory.